planarenvs/planarCommon/planarEnv.py

```diff
@@ -76,6 +85,8 @@ def reset(self, pos=None, vel=None):
     def step(self, a):
         self.action = a
         self.integrate()
+        for sensor in self._sensors:
+            self.state[sensor.name()] = sensor.sense(self.state, self._goals, self._obsts, self.t())
```

Comment:
The function call is great and exactly as I hoped it would be. Passing `self._goals` and `self._obsts` seems like the better solution here.
I would prefer to not add it to `self.state` as it might lead to trouble in the integration scheme. Could make a second attribute to the `planarEnv` class that stored sensed information? You then only concatenate them in the function `self._get_ob()`.

Reply:
Any news on that?

Reply:
@maxspahn where do you see the potential conflict? I can split it into another state of course (e.g. sensorState). However is the Sensor information not also part of the state space?

Reply:
The state is passed to the integration scheme. At some future point, we might want to check the size of the state that is passed into the continuous dynamic (or the integrated one) to raise errors if something is going wrong there. Such an assertion would fail if the size of the state is different depending on how many sensors there are. 
For me, the state is the state of the robot, although I know that this is not really true in the RL-community.

Reply:
Okay I understand. I thought it might be sufficient, since it is easy to access the relevant keys in the state now using `'x'` or `'xdot'` as it is a dictionary. 
Should I go with the option to include an extra "sensorState" variable? I would then add the "state" with the "sensorState" in the function [_get_ob()](https://github.com/alxschwrz/gym_envs_planar/blob/5727bf767cffb6bec36f807f48f68917383f5081/planarenvs/planarCommon/planarEnv.py#L101) in order to return the observation.